services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js

```diff
@@ -248,11 +305,22 @@ export class ProcessPermit extends Component {
             </h2>
             <NOWProgressActions tab="PRO" />
             {!isProcessed && (
-              <Dropdown overlay={this.menu(validationErrors)} placement="bottomLeft">
-                <Button type="secondary" className="full-mobile">
-                  Process <DownOutlined />
+              <>
```

Comment:
I'm not sure if this fragment is necessary as there is already one child "AuthorizationWrapper"?

Reply:
ah correct, I had something else there origionally.